utils/validations.tsx

```diff
@@ -530,11 +530,7 @@ export const getDualFinanceGovernanceAirdropSchema = ({
   })
 }
 
-export const getDualFinanceMerkleAirdropSchema = ({
-  form,
-}: {
-  form: any
-}) => {
+export const getDualFinanceMerkleAirdropSchema = ({ form }: { form: any }) => {
```

Comment:
if this file has only lint fixes, revert it (to make the final integration easier)